emission/storage/decorations/common_place_queries.py

```diff
@@ -0,0 +1,83 @@
+import emission.core.wrapper.common_place as ecwcp
+import emission.core.get_database as edb
+
+#################################################################################
+############################ database functions #################################
+#################################################################################
+
+def save_common_place(common_place):
+    db = edb.get_common_place_db()
+    db.save(common_place)
+    # db.insert({
+    #     "coords" : common_place.coords,
+    #     "edges" : common_place.edges,
+    #     "successors" : common_place.successors,
+    #     "user_id" : common_place.user_id
+    #     })
+
+def get_common_place_from_db(_id):
+    db = edb.get_common_place_db()
+    json_obj = db.find_one({"_id" : _id})
+    return make_common_place_from_json(json_obj)
+
+def make_common_place_from_json(json_obj):
```

Comment:
why do you even need this function? Your json_obj is already a dict, right? You should be able to just convert it directly to a CommonPlace.